src/config.js

```diff
@@ -145,6 +145,12 @@ export const conf = {
   },
 };
 
-export default (appConfig, configFiles, options={}) => convict({ ...conf, ...appConfig })
-  .load(options)
-  .loadFile(configFiles);
+
+const loadConfig = (appConfig, configFiles, options={}) => {
+  const config = convict({ ...conf, ...(appConfig || {}) }).load(options)
```

Comment:
could you not set `appConfig={}` default in argument instead of short-circuiting?